.github/workflows/continuous-integration-linux.yml

```diff
@@ -99,15 +99,6 @@ jobs:
       - name: maybe_use_flang_new
         if: ${{ matrix.cxx == 'clang++' && startsWith(matrix.distro,'fedora:') }}
         run: echo "FC=flang-new" >> $GITHUB_ENV
-      - name: maybe_use_external_gtest
-        if: ${{ matrix.distro == 'ubuntu:latest' }}
-        run: sudo apt-get update && sudo apt-get install -y libgtest-dev
-      - name: maybe_install_clang_tidy_fedora
-        if: ${{ startsWith(matrix.distro,'fedora:') }}
-        run: sudo yum update -y && sudo yum install -y clang-tools-extra
```

Comment:
I don't remember seeing `clang-tools-extra` installed

Reply:
Not in ubuntu at least
https://github.com/kokkos/ci-containers/blob/1ef260f5ddbd95c5d5e00599a56380455b454640/ubuntu#L9-L10

Reply:
Nevermind I see the if.  Sorry about the noise.

Reply:
I thought the same, but @masterleinad saw it was already installed in https://github.com/kokkos/ci-containers/pull/60.